src/app/Counters/ResetCounter.tsx

```diff
@@ -39,7 +40,7 @@ const ResetCounter = (props: {
         >
           {t('cache-managers.counters.reset-action')}
         </Button>,
-        <Button key={'Cancel'} aria-label={'Cancel'} variant={ButtonVariant.link} onClick={props.closeModal}>
+        <Button key={'Cancel'} aria-label={'Cancel'} variant={ButtonVariant.link} onClick={props.closeModal} data-cy="cancelButton">
```

Comment:
@andyuk1986 same here for the renaming please

Reply:
Done :)